theme6/source/stylesheets/site.css.scss

```diff
@@ -1,159 +1,169 @@
-      display: inline-block;
-    }
-    footer .information ul{
-      margin-top: 2%;
-      text-align: center;
-    }
-
-    footer .author {
-      display:block;
-      text-align: center;
-    }
-
-    footer .author p {
-      color: #72808C;
-      font-family:'Montserrat', sans-serif;
-      font-size: 16px;
-      line-height: 18px;
-    }
-/*=============Stylesheet MOBILE==========*/
-    @media screen and(max-width:320px) {
-      .information{display: none;}
-      .logo{display:none;}
-      #twt{
-        margin-left:-20px;
-        margin-right: -20px;}
-
-        .contenedorsocial{
-          display:flex;
-        margin-left: 50px;}
-        .box-social{align-content:center;}
-        .box-social p{font-size: 14px}
-        .boxsocialmedia{
-          width: 95%;
-          display:inline-flex;
-          justify-content:space-between;
-          align-content:center;
-        }
-
-        .contenedornews{
-          margin-top: 5px;
-          display:block;
-          align-content: center;
-          width: 100%;
-        }
-        .contenedornews p{
-          font-size: 15px;
-        }
-        .contenedornews input{
-          height: 40px;
-          width: 305px;
-          border: 2px solid #D2D2D2;
-          border-radius: 2px;
-          background-color: #FBFBFB;
-        }
-        footer .author p{
-          margin-top: 35px;
-          font-size: 12px
-        }
-    }
+footer {
+  bottom: 0;
+  display: inline;
+  width: 100%;
+}
+
+.information {
+  display: inline-block;
+  justify-content: center;
+  margin-top: 20px;
+  text-align: center;
+  width: 100%;
+}
+
+li {
+  display: inline;
+}
+
+.social-media {
+  color: #72808C;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  line-height: 21px;
+}
+
+.sm {
+  border: 2px solid #72808c;
+  border-radius: 3px;
+  height: 35px;
+  width: 93px;
+}
+
+.fb {
+  background: url("../images/fb.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  margin-right: 23px;
+}
+
+.twt {
+  background: url("../images/twt.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+}
+
+.g {
+  background: url("../images/g+.svg");
+  background-position: center center;
+  background-repeat: no-repeat;
+  margin-left: 23px;
+}
+
+.newsletter {
+  color: #72808c;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 18px;
+  height: 35px;
+  justify-content: center;
+  line-height: 21px;
+  text-align: center;
+  width: 300px;
+}
+
+.social-media-box {
+  align-content: center;
+  text-align: center;
+  width: 350px;
+}
+
+.social-media-icon-group button {
+  cursor: pointer;
+}
+
+.social-media-container {
+  align-items: center;
+  display: inline-flex;
+  justify-content: center;
+  width: 66%;
+}
+
+.logo {
+  margin-left: 96px;
+  margin-right: 96px;
+}
+
+.news-container {
+  display: inline-flex;
+  width: 33%;
+}
+
+.information a {
+  align-items: center;
+  color: #72808C;
```

Comment:
Color literals like `#72808C` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#72808C` should be written as `#72808c`